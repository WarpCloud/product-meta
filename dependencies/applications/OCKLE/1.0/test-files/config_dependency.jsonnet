{
  application_type: "OCKLE",
  application_name: "ockle1",
  application_version: "1.0",
  user_config: {},
}
