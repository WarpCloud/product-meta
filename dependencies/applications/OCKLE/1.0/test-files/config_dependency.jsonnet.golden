{
   "Customized_Namespace": "default",
   "Develop": false,
   "TCU": {
      "ockle": {
         "c": 0.5,
         "m": 0.5,
         "s": 0.5
      }
   },
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "ockle1",
            "application_type": "OCKLE"
         },
         "configs": {
            "ockle_cpu_limit": 1,
            "ockle_cpu_request": 1,
            "ockle_memory_limit": 4,
            "ockle_memory_request": 4
         },
         "dependencies": [
            {
               "moduleName": "txsql",
               "name": "txsql"
            }
         ],
         "instance_settings": { },
         "moduleName": "ockle",
         "name": "ockle1-ockle",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
